types/app.d.ts

```diff
@@ -223,14 +269,18 @@ declare namespace Realm {
         /**
          * All authenticated users.
          */
-        readonly allUsers: Readonly<User<FunctionsFactoryType, CustomDataType>[]>;
+        readonly allUsers: Readonly<
```

Comment:
Do you actually need the `Readonly<>` here in addition to the `readonly` keyword?

Reply:
The first warns if the user assigns to `allUsers` the `Readonly` warns if the user assigns to any of the elements of the list.